src/main/webapp/scripts/script.js

```diff
@@ -121,3 +123,16 @@ function displayLoading(containerId) {
   containerDom.innerText = '';
   containerDom.className = 'spinner-border text-primary';
 }
+
+/* Checks URL params for if there is a video that needs to be searched */
+function searchVideoFromParams() {
+  queryString = window.location.search;
+  if(queryString) {
+    document.getElementById('video-url').value = 'https://www.youtube.com/watch' + window.location.search;
+    analyzeVideo();
+  }
+}
```

Comment:
is this the best way to do this? No.
is it the easiest way to do it with the code we currently have and within the last week we have? Yeah :/

Reply:
thanks for clarifying :)